### PR TITLE
Adds possibility to select a default calendar

### LINKED ISSUE #175: Store state in create meeting stepper
**To Reproduce**

1. Go to meeting/create
2. Fill in the general information
3. Click next
4. Click back
5. see that all data I've previously entered is gone.

**Expected behavior**
The data I have already entered should be stored.

### PR DESCRIPTION
**Pull request by Lasklu**
closes #171 
closes #175

### CODE REVIEW

**Comment by rohansaw** [comment]
closes #175

File: app/appointments/queries/getDefaultConnectedCalendar.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import db from "db"
+
+export default async function getDefaultCalendarByUser(id: number) {
+  const defaultCalendar = await db.connectedCalendar.findFirst({
+    where: { id: id },
+  })
+
+  if (!defaultCalendar) return null //Todo handle
```
Comment (sspangenberg) [inline app/appointments/queries/getDefaultConnectedCalendar.ts:8]: Why don't you just return the dafaultCalendar for every case. As far as I can see you are checking the result in e.g. createAppointment anyway...

File: app/auth/pages/login.tsx [hunk 2]
```diff
@@ -55,19 +55,18 @@ const LoginPage: BlitzPage = () => {
...
-            <Button id="login" variant="primary" onClick={processLogin}>
+          <Col xs={8}>
+            Not registered yet?{" "}
+            <Link href="/signup">
+              <a className="text-nowrap">Sign up</a>
+            </Link>{" "}
+            now!
+          </Col>
+          <Col xs={4} className="d-flex justify-content-end">
+            <Button id="login" variant="success" onClick={processLogin}>
```
Comment (sspangenberg) [inline app/auth/pages/login.tsx:66]: I think a green button does not fit that well, why aren't we simply using the default orange button here?

File: app/auth/pages/signup.tsx [hunk 3]
```diff
@@ -77,18 +77,17 @@ const SignupPage: BlitzPage = () => {
...
+            Already registered?{" "}
             <Link href="/login">
-              <Button variant="outline-primary">Log In</Button>
-            </Link>
+              <a className="text-nowrap">Sign in</a>
+            </Link>{" "}
+            now!
+          </Col>
+          <Col xs={4} className="d-flex justify-content-end">
+            <Button id="login" variant="success" onClick={processSignup}>
```
Comment (sspangenberg) [inline app/auth/pages/signup.tsx:88]: Same as login

File: app/users/queries/getDefaultCalendarByUser.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+export default async function getDefaultCalendarByUser(_ = null, ctx: Ctx) {
+  ctx.session.authorize()
+
+  const defaultCalendar = await db.defaultCalendar.findFirst({
+    where: { userId: ctx.session!.userId },
+  })
+
+  if (!defaultCalendar) return null
+  return defaultCalendar.calendarId
```
Comment (sspangenberg) [inline app/users/queries/getDefaultCalendarByUser.ts:12]: Would be cleaner if we return the calendar here and check or access the id in the requesting method

File: app/auth/pages/login.tsx (hunk 2 above)
Comment (rohansaw) [reply app/auth/pages/login.tsx:72]: I thought this might distinguish better between the login and signup, but can also change

**Review by sspangenberg — APPROVED** [review]

**Pull request merged**